### ISSUE TITLE
WC 3.6 Warning in Woo-wallet Rest API

### ISSUE THREAD

**Issue by plonknimbuzz** [malsubrata/woo-wallet#32]
hi @malsubrata 
i just want to report issue,
yesterday i updated my WC from 3.5.8 to 3.6
after that i got this warning message.
this only happen in 3.6 only
```
Notice: wc_rest_controller was called incorrectly. Classes that extend the WooCommerce/WordPress REST API should only be loaded during the rest_api_init action, or should call WC()->api->rest_api_includes() manually. Backtrace: require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, WooCommerce->on_plugins_loaded, do_action('woocommerce_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, WooWallet->woocommerce_loaded_callback, WooWallet_API->__construct, WooWallet_API->rest_api_init, WooWallet_API->rest_api_includes, include_once('/plugins/woo-wallet/includes/api/class-wc-rest-woo-wallet-controller.php'), spl_autoload_call, WC_Autoloader->autoload, wc_doing_it_wrong Please see Debugging in WordPress for more information. (This message was added in version 3.6.) in /var/www/html/tasimportir/wp-includes/functions.php on line 4667
```

you can see it here: https://github.com/woocommerce/woocommerce/blob/master/includes/class-wc-autoloader.php#L94

thanks

**Comment by alanafoleykeane** [comment]
HI @plonknimbuzz 

I am experiencing the same issue after updating to **WooCommerce 3.6.1 version**. 

`Classes that extend the WooCommerce/WordPress REST API should only be loaded during the rest_api_init action, or should call WC()->api->rest_api_includes() manually`

_Can you advise on how you resolved this issue please?_

Thank you in advance, 
Alana

**Comment by malsubrata** [comment]
We have fixed this issue in our plugin version 1.3.6 please update and let us know.
